### PR TITLE
Update supported apps

### PR DESCRIPTION
**Pull request by MuhammetEminCakir**

### CODE REVIEW

File: docs/GSMTC-Support-And-Popular-Apps.md [hunk 1]
```diff
@@ -38,9 +38,13 @@ Users tend to know which functionalities are supported by ModernFlyouts' Media S
 | Movies & TV    | 🟢 | 🟩 | 🟩 | 🟩 | 🟪 | 🟩 | 🟪 | 🟩 | 🟥 | 🟥 | 🟩 | 🟩 |
 | [myTube Beta](https://www.microsoft.com/store/productId/9WZDNCRDT29J) | 🟢 | 🟩 | 🟩 | 🟩 | 🟥 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 | 🟩 | 🟥 |
 |**AUDIO**|**AND**|**VIDEO**|
-| [VLC (UWP)](https://www.microsoft.com/store/productId/9NBLGGH4VVNH) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 | 🟩 | 🟥
+| [VLC (UWP)](https://www.microsoft.com/store/productId/9NBLGGH4VVNH) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 | 🟩 | 🟥 |
+| [MPC-BE](https://sourceforge.net/projects/mpcbe/) | 🟢 | 🟩 | 🟩 | 🟩 | 🟥 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 | 🟥 | 🟥 |
+| [MPV](https://mpv.io/installation/) | 🟡 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 |
 | [VLC (Desktop)](https://www.videolan.org/vlc/) | 🔴 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 |
```
Comment (Samuel12321) [inline docs/GSMTC-Support-And-Popular-Apps.md:44]: Perhaps reorder to keep vlc uwp and desktop together

**Review by Samuel12321 — COMMENTED** [review]
Hi @MuhammetEminCakir thanks so much for the pr, would you mind re ordering as I suggested in the comments.

**Comment by Samuel12321** [comment]
This will probably be merged directly into the 0.11-final branch.

**Comment by MuhammetEminCakir** [comment]
@ShankarBUS 
I found a plugin for VLC.

**Comment by MuhammetEminCakir** [comment]
@Samuel12321 
What are you thinking about design of MPV's plugins informafion

**Comment by MuhammetEminCakir** [comment]
image
Can i delete this section? Because VLC(Desktop) already supports SMTC with an plugin.
@Samuel12321

**Comment by Samuel12321** [comment]
No, that is a link to a request for build in support (without plugin)

**Comment by MuhammetEminCakir** [comment]
Can i add a "Others" category for Discord etc?

**Comment by MuhammetEminCakir** [comment]
I'm not sure. But pretty sure Discord isn't a media player. Maybe WhatsApp, Telegram and android emulators

**Comment by Cyberdroid1** [comment]
I feel, we will keep it limited to media apps, since that is the main purpose of the media flyout.

@Samuel12321 what do you think?

**Comment by MuhammetEminCakir** [comment]
should i remove the Discord?

**Comment by Cyberdroid1** [comment]
That would be better, yes.

**Review by Samuel12321 — COMMENTED** [review]
Great progress, thanks so much for your help

Just noticed Media Player (grove) is incorrect, it doesnt show media title

File: docs/GSMTC-Support-And-Popular-Apps.md [hunk 2]
```diff
@@ -38,9 +38,13 @@ Users tend to know which functionalities are supported by ModernFlyouts' Media S
...
 | [PotPlayer](https://daumpotplayer.com/download/) | 🔴 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 |
+| [MPC-HC (clsid2)](https://github.com/clsid2/mpc-hc) | 🔴 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 |
+| [Amazon Prime Video](https://www.microsoft.com/store/productId/9P6RC76MSMMJ) | 🔴 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 | 🟥 |
```
[the remark this answers is not in the document]
Comment (MuhammetEminCakir) [reply docs/GSMTC-Support-And-Popular-Apps.md]: I don't understand. Should i move above mpv or mpc?

**Review by Samuel12321 — APPROVED** [review]

**Comment by Samuel12321** [comment]
Looking Great, thanks for all your help @MuhammetEminCakir 
Was going to wait till 0.11 release to merge, but as it is an important document that has been significantly updated will merge it now.
Any further improvements just open another PR

**Pull request merged**